Tests/mocks/Operands.h

```diff
@@ -0,0 +1,850 @@
+#pragma once
```

Comment:
"Operands.h" doesn't sound clear. If these classes relate to ccec, it can be "ccec.h", or a more appropriate name.  There is a pr with conflicting change https://github.com/rdkcentral/rdkservices/pull/3710/files#diff-f57c4c7c1defa058360a77be6fbc5dadbb9e2d19ccb354da66e7156ab9252ecc . I think we need only one variant of these classes.

Reply:
Modified "Operands.h" as "HdmiCec.h" , For #3710  also will use this "HdmiCec.h" in moving forward got confirmation with [hgfell683] .